### PR TITLE
[OpenVINO] Update models for tests 

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Replace model at tests from OMZ models to analogs at `tests/torch/test_models`.
All graph, scales references were updated accordingly.
New dependency for OV tests were added: `timm`, `addict`, `efficientnet_pytorch`

### Reason for changes

To vanish places of OMZ usage.

### Related tickets

136314

### Tests

All references were updated

### CODE REVIEW

File: tests/openvino/native/quantization/test_fq_params_calculation.py [hunk 1]
```diff
@@ -115,32 +116,25 @@ def test_overflow_fix_scales(overflow_fix):
...
 
-    ref_stats_name = str(Path(model_path).name).rsplit(".", maxsplit=1)[0] + f"_{preset.value}.json"
+    quantized_model = quantize_model(ov_model, {"preset": preset, "inplace_statistics": inplace_statistics})
+    nodes = get_fq_nodes_stats_algo(quantized_model)
+    ref_stats_name = model_name + f"_{preset.value}.json"
     ref_stats_path = get_actual_reference_for_current_openvino(REFERENCE_SCALES_DIR / ref_stats_name)
 
     # Uncomment lines below to generate reference for new models.
     # from tests.shared.helpers import dump_to_json
+
```
Comment (KodiaqQ) [inline tests/openvino/native/quantization/test_fq_params_calculation.py:137]: Minor. Let's keep it as is.

File: tests/openvino/requirements.txt [hunk 2]
```diff
@@ -9,3 +9,6 @@ pytest-xdist
 torch
 torchvision
 virtualenv
+addict>=2.4.0
+timm==0.9.2
+efficientnet_pytorch==0.7.1
```
Comment (KodiaqQ) [inline tests/openvino/requirements.txt:14]: Minor. The new line was missed.

**Review by KodiaqQ — APPROVED** [review]

File: tests/openvino/requirements.txt (hunk 2 above)
Comment (kshpv) [reply tests/openvino/requirements.txt]: done

File: tests/openvino/native/quantization/test_fq_params_calculation.py (hunk 1 above)
Comment (kshpv) [reply tests/openvino/native/quantization/test_fq_params_calculation.py]: done

**Pull request merged**